smart-cards/src/main/resources/application.properties

```diff
@@ -2,9 +2,16 @@ data-query.internal-url=unset
 data-query.r4-base-path=unset
 jwk-set.private-json=unset
 jwk-set.current-key-id=unset
+metadata.security.token-endpoint=unset
+metadata.security.authorize-endpoint=unset
+metadata.security.management-endpoint=unset
+metadata.security.revocation-endpoint=unset
 public-r4-base-path=unset
 public-url=unset
 server.port=8096
 spring.autoconfigure.exclude=gov.va.api.health.autoconfig.configuration.SecureRestTemplateConfig
 ssl.enable-client=false
 web-exception-key=unset
+well-known.capabilities=unset
+well-known.response=unset
+well-known.scopes-supported=unset
```

Comment:
Are these different per environment? If not, we can set them all here and not deal with DU.

Reply:
👍 and I'll take it a step further: if these values are the same for all environments, skip app properties altogether and hardcode them directly in `WellKnownController`